examples/create-setters/simple/README.md

```diff
@@ -9,6 +9,7 @@ In this example, we will see how to add setter comments declaratively to
 resource fields using `create-setters` function.
```

Comment:
The create-setters KRM config function adds comments to resource fields with setter references. => The create-setters KRM config function adds setter comments to resource fields to be parameterized.